model/registry/system.yaml

```diff
@@ -65,9 +65,6 @@ groups:
             - id: free
               value: 'free'
               stability: experimental
-            - id: shared
```

Comment:
it's no longer possible to remove enum members (as it breaks code generated semconv) and you have to deprecate it instead.